pinot-common/src/main/java/org/apache/pinot/common/request/context/LiteralContext.java

```diff
@@ -136,6 +137,11 @@ public LiteralContext(Literal literal) {
         }
         _value = literal.getFieldValue().toString();
         break;
+      case BINARY_VALUE:
+        _type = FieldSpec.DataType.BYTES;
+        _value = ((ByteBuffer) literal.getFieldValue()).array();
+        _bigDecimalValue = BigDecimal.ZERO;
```

Comment:
Out of scope of this PR, does thrift support big decimal? Should we also introduce it?

Reply:
yes, added one more commit for this, please let me know if you want to apply it